R/whatNWISData.r

```diff
@@ -57,52 +50,48 @@
 #' @importFrom lubridate parse_date_time
 #' @examples
 #' \dontrun{
-#' availableData <- whatNWISdata('05114000')
+#' availableData <- whatNWISdata(siteNumber = '05114000')
 #' # To find just unit value ('instantaneous') data:
-#' uvData <- whatNWISdata('05114000',service="uv")
-#' uvDataMulti <- whatNWISdata(c('05114000','09423350'),service=c("uv","dv"))
-#' site_ids <- c("01491000","01645000")
-#' flowAndTemp <- whatNWISdata(site_ids, parameterCd=c("00060","00010"))
+#' uvData <- whatNWISdata(siteNumber = '05114000',service="uv")
+#' uvDataMulti <- whatNWISdata(siteNumber = c('05114000','09423350'),service=c("uv","dv"))
+#' flowAndTemp <- whatNWISdata(stateCd = "WI", service = "uv", parameterCd = c("00060","00010"))
 #' }
-whatNWISdata <- function(siteNumbers,service="all",parameterCd="all",statCd="all"){
+whatNWISdata <- function(...){
```

Comment:
I was originally leaving in the previous args so that it wouldn't break people's code that use this function. Think that will be an issue?